server/app/routes/product.$productId.tsx

```diff
@@ -283,7 +283,12 @@ export default function Route() {
                   }}
                 >
                   <span className="w-full">
-                    {index + 1}. {issue.text}
+                    {index + 1}.{" "}
+                    {issue.classification &&
+                    issue.classification !== "UNKNOWN_ISSUE" ? (
+                      <b>{issue.classification}:</b>
```

Comment:
The classification is more bold than the card title
```suggestion
                      <span className="font-semibold">{issue.classification}:</span>
```